### PR TITLE
build: update go to v1.19.4-beta

### PR DESCRIPTION
**Pull request by positiveblue**
- Since Go v1.19.0 `go fmt` also formats commented code so files need to be reformated.
- Includes the fix for the lint complains about vsc

P.S: I can extract some of the steps in the actions in a comment one for  `Seting up Golang environment` if needed in this PR too

### CODE REVIEW

File: poolscript/script.go [hunk 1]
```diff
@@ -277,7 +277,9 @@ func (r *RecoveryHelper) LocateAnyOutput(expiry uint32,
 // For version 0 (p2wsh) this returns the hash of the following script:
 // <trader_key> OP_CHECKSIGVERIFY
 // <auctioneer_key> OP_CHECKSIG OP_IFDUP OP_NOTIF
+//
```
Comment (guggero) [inline poolscript/script.go:280]: Can we indent the whole script here so it can all be formatted as a single code block?

**Review by guggero — APPROVED** [review]
LGTM, thanks for the fixes!

File: poolscript/script.go [hunk 2]
```diff
@@ -275,10 +275,13 @@ func (r *RecoveryHelper) LocateAnyOutput(expiry uint32,
...
 //
 // For version 0 (p2wsh) this returns the hash of the following script:
-// <trader_key> OP_CHECKSIGVERIFY
-// <auctioneer_key> OP_CHECKSIG OP_IFDUP OP_NOTIF
-//	<account_expiry> OP_CHECKLOCKTIMEVERIFY
-// OP_ENDIF
+//
+//	     <trader_key> OP_CHECKSIGVERIFY
+//	     <auctioneer_key> OP_CHECKSIG OP_IFDUP OP_NOTIF
+//
```
Comment (guggero) [inline poolscript/script.go:281]: We can now remove the extra newlines inserted by the formatter previously.

**Review by guggero — APPROVED** [review]

File: tools/Dockerfile [hunk 3]
```diff
@@ -11,6 +11,9 @@ RUN cd /tmp \
   && mkdir -p /tmp/build/.modcache \
   && cd /tmp/tools \
   && go install -trimpath -tags=tools github.com/golangci/golangci-lint/cmd/golangci-lint \
-  && chmod -R 777 /tmp/build/
+  && chmod -R 777 /tmp/build/ \
+  && git config --global --add safe.directory /build
```
Comment (ffranr) [inline tools/Dockerfile:15]: Is it possible to remove this line now that go has been updated?

File: clientdb/batch_snapshot.go [hunk 4]
```diff
@@ -15,25 +15,26 @@ import (
...
-//         |              |        |
-//         |             ...      ...
-//         |
-//         |-- batch-snapshot-batchid-index-bucket
-//                       |
-//                       |-- <batch id>: <sequence-num>
-//                       |-- <batch id>: <sequence-num>
-//                       |
-//                      ...
+//
```
Comment (ffranr) [inline clientdb/batch_snapshot.go:18]: nit: is this empty line intentional?

**Review by ffranr — APPROVED** [review]
Looks good!

Added one nit comment and one question.

File: tools/Dockerfile (hunk 3 above)
Comment (positiveblue) [reply tools/Dockerfile:15]: I thought yes, but no, it is not possible.

Comment (guggero) [reply tools/Dockerfile:15]: Can you try if removing the `-buster` from the docker image changes anything?

Comment (positiveblue) [reply tools/Dockerfile]: :dart:

File: clientdb/batch_snapshot.go (hunk 4 above)
Comment (positiveblue) [reply clientdb/batch_snapshot.go:18]: yes, the comment formatted adds it automatically

**Comment by guggero** [comment]
Looks like we need to bump the version of `protobuf` as the new alpine probably ships with a newer version.

**Pull request merged**